src/main/java/com/stripe/model/Invoice.java

```diff
@@ -488,7 +494,12 @@ public Invoice pay(String apiKey) throws AuthenticationException,
 	public Invoice pay(RequestOptions options) throws AuthenticationException,
```

Comment:
I think maybe the one thing that we should do here is deprecate this usage since it's inconsistent with other method calls now. I'll take care of it in master.